eslint.config.mjs

```diff
@@ -2,15 +2,13 @@
 import tsEslint from 'typescript-eslint';
 import jsEslint from '@eslint/js';
 import eslintPluginPrettierRecommended from 'eslint-plugin-prettier/recommended';
-import panda from '@pandacss/eslint-plugin';
```

Comment:
In the root eslint config I removed some of the plugins and rules that are not needed now.